src/app/qgsrasterlayerproperties.cpp

```diff
@@ -74,6 +75,7 @@
 #include <QMouseEvent>
 #include <QVector>
 #include <QUrl>
+#include <QWebFrame>
```

Comment:
I think this should be `#ifdef`'d as well

Reply:
I probably don't need it anymore, it's a leftover from a previous attempt to adapt content size to parent (now done better with CSS).